unit_tests/ut_h_react_explain.amlg

```diff
@@ -439,6 +439,34 @@
 
 	(call exit_if_failures (assoc msg "Lv4 feature residuals"))
 
+;VERIFY PREDICTION STATS
+	(assign (assoc
+		result
+			(get
+				(call_entity "howso" "react" (assoc
+					action_features (list (last iris_features))
+					context_features (trunc iris_features)
+					context_values (list 6.9 3.2 4.7 1.4)
+					details
+						(assoc "prediction_stats" (true))
+				))
+				"payload"
+			)
+	))
+
+	(print "React Lv4 local prediction stats returned: ")
+	(call assert_approximate (assoc
+		exp 0.89
+		obs (get result (list "prediction_stats" "accuracy" "E"))
+		percent 0.5
```

Comment:
might be worth trying to use the `thresh` parameter to assert_approximate rather than `percent`. I think `thresh` lets you set just a constant value that the difference cannot be greater than to pass.

I also think it seems like a better fit for the assertion below.